data/src/main/java/com/moonpig/data/ProductDetailStubRepository.kt

```diff
@@ -1,15 +1,24 @@
 package com.moonpig.data
 
-import com.moonpig.mvisample.domain.AddProductRequest
-import com.moonpig.mvisample.domain.ProductDetailEntity
-import com.moonpig.mvisample.domain.ProductDetailRepository
-import com.moonpig.mvisample.domain.RepositoryState
+import com.moonpig.mvisample.domain.entities.ProductDetail
+import com.moonpig.mvisample.domain.productdetail.AddProductRequest
+import com.moonpig.mvisample.domain.productdetail.ProductDetailRepository
+import com.moonpig.mvisample.domain.productdetail.RepositoryState
 import io.reactivex.Observable
 
 class ProductDetailStubRepository : ProductDetailRepository {
     override fun getProductDetails(): Observable<RepositoryState.GetProductDetail> =
-            Observable.just(RepositoryState.GetProductDetail.InFlight,
-                            RepositoryState.GetProductDetail.Success(ProductDetailEntity("Advanced cat fighting gift set", "Everything your kitty needs to seriously up it's fighting game! Accessories including the pictured mace not included.", 299)))
+            Observable.just(
+                    RepositoryState.GetProductDetail.InFlight,
+                    RepositoryState.GetProductDetail.Success(
+                            ProductDetail(
+                                    "Advanced cat fighting gift set",
+                                    "Everything your kitty needs to seriously up it's fighting game! Accessories including the pictured mace not included.",
+                                    299,
+                                    "https://d1xkhapf8f3lxw.cloudfront.net/uk/globalassets/gadgets-novelty/update-2/for-him_276x276.jpg?version=4a8967"
```

Comment:
Can you find a picture of an advanced cat fighting gift set to go here? :trollface: